app/src/main/java/com/kickstarter/libs/DeviceRegistrar.java

```diff
@@ -29,7 +31,13 @@ public void registerDevice() {
       return;
     }
 
-    this.context.startService(new Intent(this.context, RegisterService.class));
+    final FirebaseJobDispatcher dispatcher = new FirebaseJobDispatcher(new GooglePlayDriver(this.context));
```

Comment:
I'm also wondering if it's expense to keep making new `FirebaseJobDispatcher` objects and we should make it a singleton instead.

Reply:
I'll make a file-level function for it!